### PR TITLE
Update RB notebooks

### PR DESCRIPTION
**Pull request by kylegulshen**

### CODE REVIEW

**Comment by joshcombes** [comment]
will partially close https://github.com/rigetti/forest-benchmarking/issues/98

**Comment by joshcombes** [comment]
I pulled the branch and ran locally; it ran quickly!

The main suggestion I have is to qualify the error per gate statement at the start. Make it clear that you are referring to error per Clifford and will use the phrase error per gate in that context.

**Comment by joshcombes** [comment]
Hey Kyle, can you check the run times of the RB variants in the notebook tests. If they are taking too long either skip specific cells using `# NBVAL_SKIP` or using pytest.mark.slow. I think the mark.slow only skips for local tests so it can still make the remote build take a long time.

**Comment by joshcombes** [comment]
@kylegulshen The test passes on gitlab. I can't tell if the failed test on semaphoreci is legit or not. It failed twice with the same error.

File: examples/randomized_benchmarking.ipynb [hunk 1]
```diff
@@ -0,0 +1,449 @@
...
+    "\n",
+    "- Generate many random sequences for each depth\n",
+    "\n",
+    "- Estimate the 'survival probability' for each sequence by taking the fraction of outcomes matching the initial state over many shots. Here we use the `ObservablesExperiment` framework which estimates observable expectation values from which we can calculate the survival probability.\n",
+    "\n",
+    "- Fit an exponential decay model to the estimated survival probabilities. \n",
+    "\n",
+    "- Extract the decay parameter from the fit and convert to 'average gate error'\n",
+    "\n",
+    "Note that the interpretation of RB it is an active field of research and see [Wallman](https://arxiv.org/abs/1703.09835v4) and [Proctor et al.](https://arxiv.org/abs/1702.01853) for more information.\n",
```
Comment (kylegulshen) [inline examples/randomized_benchmarking.ipynb]:
```suggestion
    "Note that the interpretation of RB is an active field of research; see [Wallman](https://arxiv.org/abs/1703.09835v4) and [Proctor et al.](https://arxiv.org/abs/1702.01853) for more information.\n",
```

**Pull request merged**